client/src/components/Followers/style.scss

```diff
@@ -0,0 +1,13 @@
+.modal-body {
+  height: 25em;
+  max-height: 100%;
+  overflow-y: auto;
+}
+
+.modal-header {
+  padding: 0.7em;
+}
+
+h5,p {
+  margin-bottom: 0;
+}
```

Comment:
Where is your wrapper for scss?

Reply:
There are two wrappers: profile and followers-modal.